Tests/WebApi.UnitTests/Applications/ActorOperations/Commands/DeleteActor/DeleteActorCommandValidatorTest.cs

```diff
@@ -0,0 +1,53 @@
+using FluentAssertions;
+using MovieStoreWebApi.Applications.ActorOperations.Commands.DeleteActor;
+using MovieStoreWebApi.Applications.ActorOperations.Commands.UpdateActor;
+using MovieStoreWebApi.DbOperations;
+using MovieStoreWebApi.Entities;
+using Tests.WebApi.UnitTests.TestSetup;
+using Xunit;
+
+namespace Tests.WebApi.UnitTests.Applications.ActorOperations.Commands.DeleteActor
+{
+    public class DeleteActorCommandValidatorTest : IClassFixture<CommonTestFixture>
+    {
+        private readonly MovieStoreDbContext _context;
+        public DeleteActorCommandValidatorTest(CommonTestFixture testFixture)
+        {
+            _context = testFixture.Context;
+        }
+
+        [InlineData(0)]
+        [InlineData(-1)]
+        [InlineData(-2)]
+
+        [Theory]
+        public void WhenInvalidInputAreGiven_Validator_ShouldBeReturnErrors(int ActorId)
+        {
+            DeleteActorCommand command = new DeleteActorCommand(null);
```

Comment:
Delete comman'dine context verilmemis burada, bu sekildeyken testin amaci yerine getirilmemis oluyor.

Reply:
command'a context yerine null verilmesinin sebebi testin teorik olmasından dolayı input olarak InlineData'dan parametre almasıdır.

Reply:
bu testte amac validator'in commandi validate etmesi, command zaten calistirilmiyor yani